manifests/init.pp

```diff
@@ -416,6 +420,9 @@
     class { 'pulpcore::plugin::rpm':
       use_pulp2_content_route => $proxy_pulp_yum_to_pulpcore,
     }
+    class { 'pulpcore::plugin::deb':
+      use_pulp2_content_route => $proxy_pulp_deb_to_pulpcore,
```

Comment:
This requires a corresponding PR to https://github.com/theforeman/puppet-pulpcore/blob/master/manifests/plugin/deb.pp to allow this.

Reply:
ahh thanks!